project.clj

```diff
@@ -7,11 +7,17 @@
                  [prismatic/schema "1.1.6"]]
   :profiles {:dev {:dependencies [[org.clojure/math.combinatorics "0.1.3"]
                                   [org.clojure/data.fressian "0.2.1"]]}
-             :provided {:dependencies [[org.clojure/clojurescript "1.7.170"]]}}
-  :plugins [[lein-codox "0.9.0" :exclusions [org.clojure/clojure]]
-            [lein-javadoc "0.2.0" :exclusions [org.clojure/clojure]]
-            [lein-cljsbuild "1.1.3" :exclusions [org.clojure/clojure]]
-            [lein-figwheel "0.5.2" :exclusions [org.clojure/clojure]]]
+             :provided {:dependencies [[org.clojure/clojurescript "1.7.170"]]}
+             :recent-clj {:dependencies [^:replace [org.clojure/clojure "1.9.0-RC1"]
+                                         ^:replace [org.clojure/clojurescript "1.9.946"]]}}
+  :plugins [[lein-codox "0.10.3" :exclusions [org.clojure/clojure
+                                              org.clojure/clojurescript]]
+            [lein-javadoc "0.3.0" :exclusions [org.clojure/clojure
+                                               org.clojure/clojurescript]]
+            [lein-cljsbuild "1.1.7" :exclusions [org.clojure/clojure
+                                                 org.clojure/clojurescript]]
+            [lein-figwheel "0.5.14" :exclusions [org.clojure/clojure
+                                                 org.clojure/clojurescript]]]
```

Comment:
I should probably investigate what exclusions here are actually needed.  I just added blanket ClojureScript exclusions for all plugins.

Some old versions of the plugins had ns forms that don't compile with 1.9.  Anyway, I don't see any reason not to update these plugins to the most recent versions since I don't expect that to impact users.

Reply:
Actually on reflection this won't do any harm so I think I'll leave it.

Reply:
Yeah, I was planning to give a PR soon that refactors some of this cljs setup. There are a few things I think that could be improved. I'm not sure adding extra `:exclusions` really matters. However, you shouldn't need to exclude transitive dependencies too often if you are explicitly declaring the version of those deps you want anyways (in JDK space at least).